File: Lista pt3/Exercicio09Arrays.java
import java.util.HashSet;
import java.util.Set;

public class Exercicio09Arrays {
    public static void main(String[] args) {

        int[] array1 = {
                1, 2, 3, 4, 5,
                6, 7, 8, 9, 10
        };

        int[] array2 = {
                11, 12, 3, 14, 15,
                16, 7, 18, 19, 20
        };

        Set<Integer> numerosIguais =
                new HashSet<>();

        for (int numero1 : array1) {

            for (int numero2 : array2) {

                if (numero1 == numero2) {
                    numerosIguais.add(numero1);
                }
            }
        }

        System.out.println(
                "Números presentes nos dois arrays:"
        );

        for (int numero : numerosIguais) {
            System.out.println(numero);
        }
    }
}

File: Lista pt3/Exercicio09Arrays.js
let array1 = [
    1, 2, 3, 4, 5,
    6, 7, 8, 9, 10
];

let array2 = [
    11, 12, 3, 14, 15,
    16, 7, 18, 19, 20
];

let numerosIguais = [];

for (let i = 0; i < array1.length; i++) {

    for (let j = 0; j < array2.length; j++) {

        if (array1[i] === array2[j]) {

            if (!numerosIguais.includes(array1[i])) {
                numerosIguais.push(array1[i]);
            }
        }
    }
}

console.log("Array 1:");
console.log(array1);

console.log("Array 2:");
console.log(array2);

console.log("Números iguais:");
console.log(numerosIguais);